language_features/rabbitmq.yml

```
@@ -26,10 +26,10 @@
     - restart rabbitmq
 
   - name: add users
-    rabbitmq_user: user={{item}} password=changeme tags=administrator,{{item}} vhost=/ configure_priv=.* write_priv=.* read_priv=.* state=present
+    rabbitmq_user: user={{item.username}} password={{item.password}} tags=administrator,{{item}} vhost=/ configure_priv=.* write_priv=.* read_priv=.* state=present
```

Doesn't new form of `{{item}}` break things?
```suggestion
    rabbitmq_user: user={{item.username}} password={{item.password}} tags=administrator,{{item.username}} vhost=/ configure_priv=.* write_priv=.* read_priv=.* state=present
```

---

Honestly, I'm not sure. I've not touched this in two years. If it is supposed to be item.username/password I'll trust you and commit that suggestion

---

It seems logical to keep the original value there